nominal/core.py

```diff
@@ -311,6 +322,146 @@ def _from_conjure(cls, client: NominalClient, dataset: scout_catalog.EnrichedDat
         )
 
 
+# TODO(ritwikdixit): add support for more Checklist metadata and versioning
```

Comment:
There's a few things left out here. Namely, commit # and checklist metadata - for now, we're not really letting users do anything with the checklist other than create it.